### ISSUE TITLE
User Story 22

### ISSUE THREAD

**Issue by rabe200** [rabe200/Capstone_Tarot_App#34]
As a user i want to be able to chose between different readers
So that the whole experience gets more exciting

Description

![Image](https://user-images.githubusercontent.com/115098194/233214406-3b33cb72-77c9-4a2b-8fe8-930c6b1ede41.png)

acceptance criteria
- [ ] 4 different cards with each different name and picture
- [ ] chosing one of the cards makes the selection visually noticable
- [ ]  there is a button to the next page

tasks:
- don't get lost creating many fancy new characters. just stick to 4 for the start.

**Comment by ella-blaupause** [comment]
the continue button takes you to the next page, how do I get to the previous page? Or is that intentionally not scheduled?

**Comment by rabe200** [comment]
nah, you are right. i was thinking about, that i maybe add a navbar, but am not sure about this step yet. so i just added the prev button the acceptance criteria. thank you!
